src/main.rs

```diff
@@ -20,7 +24,42 @@ use crate::hand_stats::HandStats;
 /// (2) print out the hand
 /// (3) compute and print the card stats
 fn main() {
+    let args: PokerArgs = PokerArgs::parse();
     let mut rng = rand::thread_rng();
+    if args.run_option
+    {
+        if args.cards_number == 5{
+            output_specific::<5>(rng, args.samples_number);
+        }
+        else if args.cards_number == 7{
+            output_specific::<7>(rng, args.samples_number);
+        }
+        else{
+            {
+                println!("");
+                let five_card_hand = Hand::<5>::draw(&mut rng);
+                println!("{five_card_hand}");
+                let hand_stats = HandStats::from(&five_card_hand);
+                println!("{hand_stats}");
+                let hand_score = HandScore::from(&hand_stats);
+                println!("{hand_score}");
+
+                println!("");
+                let seven_card_hand = Hand::<7>::draw(&mut rng);
+                println!("{seven_card_hand}");
+                let hand_stats = HandStats::from(&seven_card_hand);
+                println!("{hand_stats}");
+                let hand_score = HandScore::from(&hand_stats);
+                println!("{hand_score}");
+
+                // Now draw N random hands and check the stats!
+                println!("");
+                // Note: there is probably some way to deduce the type of the RNG here...
```

Comment:
Probably?  Would be neat to figure out, but not critical.

Reply:
I'm not sure -- I would need to do some research to figure it out.  How about we leave this as-is for now, and then consider cleaning it up as a follow-up PR.